man/man1/gitsh.1

```diff
@@ -239,13 +239,10 @@ branch.
 .El
 .El
 .Pp
-Unset variables will be ignored, as then are in
-.Xr sh 1 .
-For example, the commit message produced by the following command will be
-.Ic "a commit" :
-.Bd -literal -offset indent
-@ commit -m $unset "a commit"
-.Ed
+Attempting to use an unset variable will cause a command to fail. This is
+different to
```

Comment:
Huh, I didn't know that. I'm learning a lot of en-US this week: I didn't know “donut” was the en-US spelling of “doughnut” until yesterday.

I'm happy to leave this as is, I think the phrase is similar enough to be comprehensible.